### PR TITLE
avoid cyclic dependency and chrome warning: solves #71

### PR DESCRIPTION
**Pull request by bsr203**
hi. I made the changes you suggested. didn't test locally though :-) getting bit long to get started :-)

### CODE REVIEW

File: src/Collection/common.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line import/prefer-default-export
```
Comment (radex) [inline src/Collection/common.js]: missing `// @flow` at the top of the file (that should be an ESLint rule if you can find that really quick)

Comment (bsr203) [reply src/Collection/common.js]: I added 
```
 flowtype/require-valid-file-annotation:
    - 2
    - always
```

it didn't give an error on linting. But, when I save the file, it automatically insert insert flow annotation. I am not sure why it doesn't trigger error though.

Comment (bsr203) [reply src/Collection/common.js]: if you are happy with the rule, I can add it. I quickly checked the `eslint-plugin-flowtype` repo and there are few issues to go through. Sorry, I haven't used eslint for a while and don't want to spend too much time. thanks

Comment (bsr203) [reply src/Collection/common.js]: here is the link for that rule https://github.com/gajus/eslint-plugin-flowtype#eslint-plugin-flowtype-rules-require-valid-file-annotation

Comment (radex) [reply src/Collection/common.js]: Ok, nvm then. I’ll check this myself as I’d have to configure tests to be ignored. But please add @flow to this file

Comment (bsr203) [reply src/Collection/common.js]: ok done.

**Comment by radex** [comment]
thanks @bsr203

**Pull request merged**